Examples/Sudoku-examples/CLIPS-puzzles/grid4x4-p13.clp

```diff
@@ -1,74 +1,70 @@
+;;; The puzzle is:
+;;;
+;;;     7 16  8  3  12 13  *  *   *  *  *  *   *  9 14  *
+;;;     *  1  * 13   *  *  *  *   8  3  *  *  16 12  * 11
+;;;    12  * 15  *   *  5  6  *  13  4 11  *   *  *  *  3
+;;;     4  9  *  2   1  * 16  3  14 12  *  7  10  * 15 13
+;;;
+;;;     *  *  *  *   8  *  9 11  10  7  3  *   * 16  *  *
+;;;    16  4  *  1   *  7  *  *   6  * 14  *  13  * 11  9
+;;;     *  7  2  6   *  * 14  *   1  *  *  *   5  3  4  *
+;;;     *  *  * 11   6  *  *  *   4  *  * 13   *  *  *  *
+;;;
+;;;     *  *  *  *  14  *  *  9   *  *  * 10  11  *  *  *
+;;;     *  3 16 14   *  *  * 10   * 11  *  *   4 13  9  *
+;;;    10  5  *  8   * 11  *  6   *  *  4  *   7  * 12 15
+;;;     *  * 13  *   * 12  4  8   9  6  * 15   *  *  *  *
+;;;
+;;;    11  6  * 16   4  *  8  1   3 13  * 12   9  * 10  5
+;;;    13  *  *  *   *  6 12 14   * 10 16  *   * 11  *  7
+;;;     8  * 12  4   *  *  5  7   *  *  *  *   2  * 13  *
+;;;     * 10  7  *   *  *  *  *   *  *  8  6  14  4  1 12
+;;;
+;;; The solution is:
+;;;
+;;;     7 16  8  3  12 13 11 15   2  5 10  1   6  9 14  4
+;;;     5  1  6 13  10 14  7  4   8  3 15  9  16 12  2 11
+;;;    12 14 15 10   9  5  6  2  13  4 11 16   1  7  8  3
+;;;     4  9 11  2   1  8 16  3  14 12  6  7  10  5 15 13
+;;;
+;;;    14 13  5 12   8  4  9 11  10  7  3  2  15 16  6  1
+;;;    16  4  3  1   2  7 10 12   6 15 14  5  13  8 11  9
+;;;     9  7  2  6  16 15 14 13   1  8 12 11   5  3  4 10
+;;;    15  8 10 11   6  3  1  5   4 16  9 13  12 14  7  2
+;;;
+;;;     6 12  4 15  14 16  3  9   7  1 13 10  11  2  5  8
+;;;     2  3 16 14   7  1 15 10  12 11  5  8   4 13  9  6
+;;;    10  5  9  8  13 11  2  6  16 14  4  3   7  1 12 15
+;;;     1 11 13  7   5 12  4  8   9  6  2 15   3 10 16 14
+;;;
+;;;    11  6 14 16   4  2  8  1   3 13  7 12   9 15 10  5
+;;;    13  2  1  9  15  6 12 14   5 10 16  4   8 11  3  7
+;;;     8 15 12  4   3 10  5  7  11  9  1 14   2  6 13 16
+;;;     3 10  7  5  11  9 13 16  15  2  8  6  14  4  1 12
+;;;
+;;; Rules used:
+;;;
+;;;    Naked Single
+;;;    Hidden Single
+;;;    Locked Candidate Single Line
+;;;    Locked Candidate Multiple Lines
+;;;    Naked Pairs
+;;;    Hidden Pairs
+;;;    X Wing
+;;;    Naked Triples
+;;;    XY-Wing
+;;;    Swordfish
+;;;    Color Conjugate Pairs
```

Comment:
Commenting these lines is not strictly necessary, I think.  I believe that CLIPS will happily ignore them.  However, I commented them for consistency with the other examples, and to ensure that no stray comment gets inadvertently interpreted as a CLIPS command.